torch/_dynamo/variables/tensor.py

```diff
@@ -479,23 +479,17 @@ def make_const_size_variable(x, **options):
                 )
             return constant_result
         elif name == "numpy":
-            from .builder import wrap_fx_proxy_cls
-
             assert not args, "Tensor.numpy() doesn't take args."
             # TODO: support force
             if kwargs and "force" in kwargs:
                 unimplemented(f"Tensor.numpy(force={kwargs['force']})")
-            return wrap_fx_proxy_cls(
-                target_cls=NumpyNdarrayVariable,
-                tx=tx,
-                proxy=tx.output.create_proxy(
-                    "call_function",
-                    torch.detach,
-                    *proxy_args_kwargs([self], {}),
-                ),
-                example_value=None,
-                **options,
+            proxy = tx.output.create_proxy(
+                "call_function",
+                torch.detach,
+                *proxy_args_kwargs([self], {}),
             )
+            return NumpyNdarrayVariable.create(tx, proxy, **options)
```

Comment:
Is this just a refactor?

Reply:
Yeah, It was a little tougher to debug w/o this.